### PR TITLE
support integration testing on macos

### PR DESCRIPTION
**Pull request by levicook**
just some quick os detection to determine how to load the compiled library the integration tests depend on

### CODE REVIEW

**Comment by levicook** [comment]
See issue #4

**Comment by levicook** [comment]
See PR #5

File: tests/test_postgres_plugin.rs [hunk 1]
```diff
@@ -215,12 +220,14 @@ fn test_postgres_plugin() {
     solana_logger::setup_with_default(RUST_LOG_FILTER);
 
     unsafe {
-        let lib = Library::new("libsolana_accountsdb_plugin_postgres.so");
+        let mut filename = "libsolana_accountsdb_plugin_postgres.dylib";
```
Comment (lijunwangs) [inline tests/test_postgres_plugin.rs]: Initialize to "libsolana_accountsdb_plugin_postgres.so" by default

Comment (levicook) [reply tests/test_postgres_plugin.rs]: Good catch! I assumed that I had -- updating presently.

**Comment by lijunwangs** [comment]
For future -- PR, I'd suggest directly creating PR against https://github.com/solana-labs/solana-accountsdb-plugin-postgres -- that will be our official repo for this plugin. Thanks!

**Pull request merged**